pkg/sdk/logging/model/filter/concat.go

```diff
@@ -79,8 +79,8 @@ type Concat struct {
 	PartialCriStreamKey string `json:"partial_cri_stream_key,omitempty"`
 }
 
-// ## Example `Concat` filter configurations
 /*
+## Example `Concat` filter configurations
 
```

Comment:
@fekete-robert this breaks go fmt, this is why the PR is failing

We could add an empty line comment if that works

Reply:
Oh :( Then it'll probably break at some other places as well, I thought this pattern will work

Reply:
this breaks everywhere, just picked this as an example

Reply:
Ok, let's discuss IRL which pattern should we use, then I'll fix it everywhere